shared/js/background/safari-wrapper.es6.js

```diff
@@ -93,14 +93,9 @@ let mergeSavedSettings = (settings, results) => {
     return Object.assign(settings, results)
 }
 
-let injectATBScripts = () => {
-    // in Safari we can't inject a script whenever we want, so instead add it everywhere
-    // and wait for the background process to tell us when to execute it
-    safari.application.activeBrowserWindow.tabs.forEach((tab) => {
-        if (tab.url.match(/https:\/\/([^/]+\.)?duckduckgo.com/)) {
-            tab.page.dispatchMessage('getATB')
-        }
-    })
+let getDDGTabUrls = () => {
+    // we don't currently support getting ATB from install page on Safari
+    return Promise.resolve([])
 }
```

Comment:
It shouldn't be too difficult to actually return URLs from here, but I guess it's not a codepath we actually care about if Safari only does gallery installs now.